src/plugins/intel_gpu/src/plugin/ops/select.cpp

```diff
@@ -73,14 +73,12 @@ static void CreateSelectOp(Program& p, const std::shared_ptr<ngraph::op::v1::Sel
         }
```

Comment:
Dynamic shape for numpy broadcasting rule is not supported. Please fix it to support..

Reply:
done